src/Thujohn/Twitter/TwitterServiceProvider.php

```diff
@@ -62,4 +62,4 @@ public function provides()
 		return ['ttwitter'];
```

Comment:
Maybe, though changing that + the facade, still gives that error.

Though it works in my project, it only gives the error when running the `ide-helper:generate` command